frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketsList.component.tsx

```diff
@@ -49,14 +56,26 @@ export const TicketsList = ({ tickets }: TicketsListProps) => {
 		setSelectedTemplates(new Set(selectedTemplates));
 	};
 
-	const getTicketsByTemplateId = (templateId: string) => tickets.filter(({ type }) => type === templateId);
+	const filterCompleted = (ticket) => {
+		const issuePropertyStatus = get(ticket, 'properties.Status');
+		const treatmentStatus = get(ticket, 'modules.safetibase.Treatment Status');
 
-	const getFilteredTickets = () => {
-		if (selectedTemplates.size === 0) return tickets;
-		return flatMap([...selectedTemplates], getTicketsByTemplateId);
+		const isCompletedIssueProperty = [TicketStatuses.CLOSED, TicketStatuses.VOID].includes(issuePropertyStatus);
+		const isCompletedTreatmentStatus = [TreatmentStatuses.AGREED_FULLY, TreatmentStatuses.VOID].includes(treatmentStatus);
+
+		return (isCompletedIssueProperty || isCompletedTreatmentStatus) === showingCompleted;
 	};
 
-	const getTemplatesForFilter = () => templates.filter(({ _id }) => getTicketsByTemplateId(_id).length > 0);
+	const getFilteredTickets = tickets.filter((ticket) => (!selectedTemplates.size || selectedTemplates.has(ticket.type)) && filterCompleted(ticket));
```

Comment:
can we change the name of this ?
```suggestion
	const filteredTickets = tickets.filter((ticket) => (!selectedTemplates.size || selectedTemplates.has(ticket.type)) && filterCompleted(ticket));
```